av_imu_launch/package.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>av_imu_launch</name>
+  <version>0.0.0</version>
```

Comment:
Indeed, the very first time I was setting all of this up I just wanted to go with 0s until we made the first "release", which getting all of these initial PRs merged may qualify as.
Back in my day we had these version numbers automatically generated and populated given our changelog updates.
It was convenient, but I guess we don't have to go to such lengths of automation (yet).
There are multiple ways we could go about it, I don't think we will be publicising our work often, but it will be very convenient indeed to generate docker images which versions correspond to the pkg versions within them.

Some relevant reading:
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes
https://medium.com/fiverr-engineering/major-minor-patch-a5298e2e1798
https://github.com/audrow/update-ros-pkg-versions

Reply:
I was thinking about the same. It wouldn't hurt to just semi-automate something so we ensure proper versioning for the good practice